patches/master_patch.patch

```diff
@@ -76,6 +76,19 @@ index 4bb4a043cbbcb562aed8926867660547cd7f018c..769bc73607b600d549b6185a37d6b0cc
    branding_path_component = "chromium"
  }
 +}
+diff --git a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
+index fff8488778d7fb667e7683ee882c43baa2d4e94f..0317a8b0bf3d97a1449ac199905f14261b701b2e 100644
+--- a/build/config/compiler/BUILD.gn
++++ b/build/config/compiler/BUILD.gn
+@@ -1829,7 +1829,7 @@ config("default_optimization") {
+     # The NaCl IRT is a special case and always wants its own config.
+     # It gets optimized the same way regardless of the type of build.
+     configs = [ "//build/config/nacl:irt_optimize" ]
+-  } else if (is_debug) {
++  } else if (is_debug || !is_official_build) {
```

Comment:
can't we just allow independent setting of `is_debug` in boostrap?

Reply:
I thought that would become a debug build and we have so many DCHECK failure case for now. This is more like temporary solution which is merged into chromium-fork too. We can remove this once debug build is stable. @bbondy